tests/pr_test.go

```diff
@@ -94,3 +97,44 @@ func TestRunUpgrade(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestPlanValidation(t *testing.T) {
+	// Regions that support Cross Region Resiliency plan
+	validCrossRegionPlanLocations := []string{"us-south", "eu-de", "jp-tok"}
+	// Regions that don't support Cross Region Resiliency plan
+	invalidCrossRegionPlanLocations := []string{"au-syd", "jp-osa", "eu-es", "eu-gb", "ca-tor", "us-east", "br-sao"}
+
+	options := &terraform.Options{
+		TerraformDir: "../examples/basic",
+		Vars: map[string]interface{}{
+			"prefix":         "validate-plan",
+			"plan":           "cross-region-resiliency",
+			"resource_group": resourceGroup,
+		},
+		Logger:  logger.Discard,
+		Upgrade: true,
+		NoColor: true,
+	}
+
+	_, initErr := terraform.InitE(t, options)
+	assert.Nil(t, initErr, "This should not have errored")
```

Comment:
If the `InitE` fails, the test will continue which is probably not expected.

Reply:
noting here that this was resolved by wrapping the assert with an `if ... { }` block